### PR TITLE
TLC-292 register page functional tests STEP-1

### PR DESCRIPTION
**Pull request by Dobrovolsky23**

### CODE REVIEW

File: 20605/specs/regression/registerStep1Functional.spec.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+describe('REGISTER PAGE STEP-1 FUNCTIONAL PART', function() {
+  beforeEach(() => {
+    RegisterPage.open();
+  });
+  it('First name field should accept letter " A "  ', function() {
+    RegisterPage.firstNameInput.setValue(inputRegisterStep1.firstLastName1);
+    expect(RegisterPage.firstNameInput.getValue()).eq(expectedRegisterStep1.firstLastName1);
+  });
+  it('First name field should accept letter " aaaaaaaaaaaaaaaaaa "  ', function() {
```
Comment (4stuntman4) [inline 20605/specs/regression/registerStep1Functional.spec.js:13]: add empty line between all Its

File: 20605/specs/regression/registerStep1Functional.spec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    RegisterPage.lastNameInput.setValue(inputRegisterStep1.firstLastName3);
+    RegisterPage.email.setValue(inputRegisterStep1.email);
+    RegisterPage.password.setValue(inputRegisterStep1.password);
+    RegisterPage.agreementCheckBox.click();
+    expect(RegisterPage.registerBtn.isClickable()).true;
+  });
+  it('user should be able to click Log in button', function() {
+  expect(RegisterPage.loginLink.isClickable()).true;
+  });
+});
```
Comment (Edemina) [inline 20605/specs/regression/registerStep1Functional.spec.js:64]: Alex, I would suggest to add empty lines between each it and at the end.
Other than that tests look good.

File: 20605/testData/register.js [hunk 3]
```diff
@@ -5,5 +5,14 @@ const newUser = {
...
   password: 'abc123'
 };
-export {newUser}
+
+const inputRegisterStep1 ={
+  firstLastName1: "A",
+  firstLastName2: "aaaaaaaaaaaaaaaaaa",
+  firstLastName3: "a'sssa-",
+  email: "Asdfgh@mail.us",
+  password: "Azxcvb@123"
```
Comment (Edemina) [inline 20605/testData/register.js:14]: Please replace all double quotes "" with single quotes ' '

File: 20605/testResult/register.js [hunk 4]
```diff
@@ -16,5 +16,12 @@ const registerPageTestResult = {
   registerBtn: 'Register',
   loginPromptLabel: 'Already have an account? Just click',
 };
+const expectedRegisterStep1 = {
+  firstLastName1: "A",
+  firstLastName2: "aaaaaaaaaaaaaaaaaa",
+  firstLastName3: "a'sssa-",
+  email: "Asdfgh@mail.us",
+  password: "Azxcvb@123"
```
Comment (Edemina) [inline 20605/testResult/register.js:24]: Please replace all double quotes "" with single quotes ' '

File: 20605/testData/register.js (hunk 3 above)
Comment (Dobrovolsky23) [reply 20605/testData/register.js:14]: I need double quotes because I'm using apostrophe symbol in my data

**Pull request merged**